module/Database/src/Form/InstallationFunction.php

```diff
@@ -38,6 +41,24 @@ public function __construct(private readonly Translator $translator)
      */
     public function getInputFilterSpecification(): array
     {
-        return [];
+        return [
+            'name' => [
+                'required' => true,
+                'filters' => [
+                    [
+                        'name' => StringTrim::class,
+                    ],
+                ],
+                'validators' => [
+                    [
+                        'name' => StringLength::class,
+                        'options' => [
+                            'min' => 2,
+                            'max' => 32,
```

Comment:
In GH-398 you are proposing to also use this for board functions. 

`Commissaris Carrièreontwikkeling en Externe Betrekkingen der s.v. GEWIS` has a length of 71 characters; `Commissaris Carrièreontwikkeling en Externe Betrekkingen` is 56 characters.

Depending on the implementation of GH-398, I propose to raise the limit to 64 or 96 characters

Reply:
I really want to remove all installation function configuration, so nothing that you can configure from the UI. The only limit will then be whatever we approve (because if a new function is required the person requesting it can easily make a PR) and what fits in the DB field.

Reply:
Requiring a PR will also raise the bar for requesting "silly" functions.